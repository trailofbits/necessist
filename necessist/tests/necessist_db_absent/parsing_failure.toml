args = ["--no-sqlite", "--root=examples/parsing_failure"]
stderr = ""

[bin]
name = "necessist"

[fs]
cwd = "../../.."

[status]
code = 0
