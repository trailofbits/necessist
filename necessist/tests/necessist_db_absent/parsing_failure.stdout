Warning: Failed to parse "src/lib.rs": expected an expression
Silence this warning with: --allow parsing-failed
0 candidates in 0 tests in 0 test files
